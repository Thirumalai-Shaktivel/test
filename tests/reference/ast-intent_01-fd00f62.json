{
    "basename": "ast-intent_01-fd00f62",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intent_01.f90",
    "infile_hash": "ecbce2d4b01a09bee24ec92ce8be335f75903e746d23a667806bf264",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-intent_01-fd00f62.stdout",
    "stdout_hash": "2e50eec14190ac792e9d1b4e4565b0ef12e115e82efbdac7d65404a9",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}