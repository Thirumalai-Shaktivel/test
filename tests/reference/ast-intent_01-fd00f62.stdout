(TranslationUnit [(Module dflt_intent [] [] [] [(Subroutine foo [(c) (d)] [] () [] [] [] [(Declaration (AttrType TypeReal [] () None) [] [(c [] [] () None ()) (d [] [] () None ()) (e [] [] () None ()) (g [] [] () None ())])] [(= 0 e (FuncCallOrArray f [] [(() c ())] [] []) ()) (= 0 g (FuncCallOrArray f [] [(() d ())] [] []) ())] [(Function f [(x)] [(AttrType TypeReal [] () None)] () () [] [] [] [(Declaration (AttrType TypeReal [] () None) [(AttrIntent In)] [(x [] [] () None ())])] [(= 0 f (* 2 x) ()) (Print 0 () [f] ())] [])])]) (Program main [(Use [] dflt_intent [] .false.)] [] [] [(SubroutineCall 0 foo [] [(() (Real "0.0") ()) (() (Real "2.0") ())] [] ())] [])])
